src/main/java/net/mcreator/ui/modgui/VillagerProfessionGUI.java

```diff
@@ -0,0 +1,209 @@
+			professionTextureFile.addItem("");
+			mcreator.getFolderManager().getTexturesList(TextureType.VILLAGER_PROFESSION)
+					.forEach(el -> professionTextureFile.addItem(el.getName()));
+		});
+
+		subpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("villagerprofessions/profession_texture"),
+				L10N.label("elementgui.villager_profession.profession_texture")));
+		subpanel.add(PanelUtils.centerAndEastElement(professionTextureFile, importProfessionTexture, 0, 0));
+
+		page1group.addValidationElement(name);
+		page1group.addValidationElement(pointOfInterest);
+		page1group.addValidationElement(actionSound.getVTextField());
+		page1group.addValidationElement(professionTextureFile);
+
+		name.setValidator(new TextFieldValidator(name, L10N.t("elementgui.villager_profession.profession_needs_name")));
+		name.enableRealtimeValidation();
+		pointOfInterest.setValidator(new MCItemHolderValidator(pointOfInterest));
+		actionSound.getVTextField().setValidator(new TextFieldValidator(actionSound.getVTextField(),
+				L10N.t("elementgui.common.error_sound_empty_null")));
+		professionTextureFile.setValidator(() -> {
+			if (professionTextureFile.getSelectedItem() == null || professionTextureFile.getSelectedItem().equals(""))
+				return new Validator.ValidationResult(Validator.ValidationResultType.ERROR,
+						L10N.t("elementgui.villager_profession.profession_needs_texture"));
+			return Validator.ValidationResult.PASSED;
+		});
+
+		panel2.add(PanelUtils.totalCenterInPanel(subpanel));
+		panel2.setOpaque(false);
+
+		clo.setPreferredSize(new Dimension(fact, fact));
+
+		JPanel clop = new JPanel();
+		clop.setOpaque(false);
+		clop.add(clo);
+
+		JPanel panel = new JPanel(new BorderLayout(35, 35));
+		panel.add("Center", panel2);
+		panel.add("South", clop);
+		panel.setOpaque(false);
+
+		addPage(L10N.t("elementgui.common.page_properties"), panel);
+
+		if (!isEditingMode()) {
+			name.setText(modElement.getName().toUpperCase(Locale.ROOT));
+		}
+	}
+
+	private void updateProfessionTexturePreview() {
+		if (professionTextureFile.getSelectedItem() == null)
+			return;
+		File professionTexture = mcreator.getFolderManager()
+				.getVillagerProfessionTextureFileForName(professionTextureFile.getSelectedItem());
+		if (!professionTexture.isFile())
+			return;
+		ImageIcon bg = new ImageIcon(
+				ImageUtils.resize(new ImageIcon(professionTexture.getAbsolutePath()).getImage(), fact, fact));
+		clo.setIcon(ImageUtils.drawOver(bg));
+	}
+
+	@Override public void reloadDataLists() {
+		super.reloadDataLists();
+		List<File> professionFiles = mcreator.getFolderManager().getTexturesList(TextureType.VILLAGER_PROFESSION)
+				.stream().toList();
+		professionFiles.forEach(texture -> {
+			if (texture.getName().endsWith(".png"))
+				texture.getName().replace(".png", "");
+		});
+		ComboBoxUtil.updateComboBoxContents(professionTextureFile,
+				ListUtils.merge(Collections.singleton(""), professionFiles.stream().map(File::getName).toList()));
```

Comment:
```suggestion
		ComboBoxUtil.updateComboBoxContents(professionTextureFile, ListUtils.merge(Collections.singleton(""),
				mcreator.getFolderManager().getTexturesList(TextureType.VILLAGER_PROFESSION)
				.stream().map(File::getName).toList()));
```
\*You needn't to remove the extension, it is kept in other mod elements.

Reply:
Done!